src/main/java/use_cases/sales_report/SalesReporterInputData.java

```diff
@@ -34,7 +34,7 @@ public SalesReporterInputData(List<String[]> rows, String startTime, String endT
      * @return a LocalDateTime object
      */
     public static LocalDateTime stringToDateTime(String dateTime){
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
```

Comment:
yeah great change here you need to follow the same format with history use case otherwise something might go wrong.